apps/search/app.prod.Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Source
+# https://turbo.build/repo/docs/handbook/deploying-with-docker#example
+# TODO https://pnpm.io/cli/fetch
```

Comment:
pnpm can be installed with more performant way, it should also be configured to cache node modules